.github/workflows/api_file_update.yaml

```diff
@@ -17,10 +17,12 @@ jobs:
         if: github.event_name == 'pull_request' && github.event.pull_request.merged == true
       - name: Update API file
         run: |
-          cd specs
-          wget https://eaterslab.herokuapp.com/api.yaml -O api_update.yaml; then
-          rm -rf api.yaml
-          mv api_update.yaml api.yaml
+          cd ./specs
+          if wget https://eaterslab.herokuapp.com/api.yaml -O api_update.yaml; then
```

Comment:
I think it would be nice to have else case with `exit 1`. Server may be down for whatever reason and we wouldn't know that API didn't update.

Reply:
fixed